cmd/icingadb/main.go

```diff
@@ -63,7 +66,9 @@ func run() int {
 	}
 	defer db.Close()
 	{
-		logger.Info("Connecting to database")
+		logger.Infof("Connecting to database at host '%s'", net.JoinHostPort(
+			cmd.Config.Database.Host, strconv.FormatInt(int64(cmd.Config.Database.Port), 10)),
```

Comment:
`fmt.Sprint(cmd.Config.Database.Port)` would do the same here but is shorter/easier to read.